ui/selectors/selectors.js

```diff
@@ -667,6 +667,11 @@ export function getShowTestNetworks(state) {
   return Boolean(showTestNetworks);
 }
 
+export function getPetnamesEnabled(state) {
```

Comment:
Minor, but is it worth a unit test for coverage? I think historically the UI is a little lacking in unit tests.

Reply:
Good idea. Done.